### DISCUSSION TITLE
Disable automatic deployment link in GitHub commit website field

### DISCUSSION THREAD

**Discussion by heltonricardo** [vercel/community#2476] [Idea]
Hello,

I would like to suggest the option to disable the automatic inclusion of the deployment link in the "website" field of GitHub commits when using Vercel for deployment. Currently, whenever a deployment is made using Vercel, the deployment link is automatically added to the "website" field in the commit, which can be inconvenient for certain use cases.

Having the ability to control the "website" field manually would provide more flexibility and control over the commit messages. It would allow users to add custom website links or choose not to include any link at all, depending on their specific requirements.

I understand that the integration between Vercel and GitHub is meant to streamline the deployment process, but having an option to disable this automation would be beneficial for those who prefer to manage the "website" field manually.

Thank you for considering this suggestion. I believe it would provide an additional level of customization and flexibility for users of Vercel and GitHub.

Best regards,
Helton.

**Comment by amyegan** [comment]
We have a [`github.silent`](https://vercel.com/docs/concepts/projects/project-configuration/git-configuration#github.silent) configuration option that stops the GitHub integration bot from commenting on pull requests and commits. Would that work for you?

**Reply by heltonricardo** [reply]
Thank you for your suggestion! Unfortunately, the `github.silent` option mentioned is specifically designed to disable comments from the GitHub integration bot on pull requests and commits. It does not directly address the automatic inclusion of the deployment link in the "website" field.

**Reply by amyegan** [reply]
I misunderstood the first time around, but I see what you mean now. Thanks for clarifying! This is a great suggestion and I'm sharing it with the team.

**Reply by amyegan** [reply]
I got some more info from the team that manages the integration and tried a few things myself. The integration only fills the "website" field  in the "About" section if it is blank. So if you fill the field with your preferred URL, the value will not be overwritten. Including a screenshot to make sure we're talking about the same thing:

<img src="https://github.com/vercel/community/assets/8247105/6fe9dc90-489a-4024-85da-cf557d6fb61f" alt="Website field in GitHub repo About section" />

**Reply by heltonricardo** [reply]
Thank you for your response and the additional information provided. I understand that the integration only fills the "website" field in the "About" section if it is blank. However, the request is to have the option to leave the field without any URL. I sincerely appreciate your assistance and kindly request considering the suggestion to include an option to disable the automatic inclusion of the deployment link. Thank you for your attention and consideration.
